CHANGELOG.md

```diff
@@ -10,6 +10,7 @@ Mapbox welcomes participation and contributions from everyone.
 * Avoid tiles re-layout on enabling terrain with zero exaggeration. ([1791](https://github.com/mapbox/mapbox-maps-android/pull/1791))
 * Asynchronous GeoJSON data parsing when adding a new style source. ([1791](https://github.com/mapbox/mapbox-maps-android/pull/1791))
 * Unify the `margin`/`translation` `Widget` APIs into the `WidgetPosition.offset`, rename `WidgetPosition.horizontal`/`WidgetPosition.vertical` to `WidgetPosition.horizontalAlignment`/`WidgetPosition.verticalAlignment`; Deprecate the original constructors and `setTranslation` APIs. ([1782](https://github.com/mapbox/mapbox-maps-android/pull/1782))
+* Added API to enable/disable rendering of world copies in mercator mode. ([1794](https://github.com/mapbox/mapbox-maps-android/pull/1794))
```

Comment:
here we're stating that it's mercator - so I assume it's not supporting globe then? I don't see anything about it in javadoc though. so the question is - does this work in globe projection and if it does not - should we handle that case?

Reply:
@kiryldz do we have world copies in globe ? I think in globe this is not the case.

Reply:
tested it with GlobeActivity. nothing critical didn't happen, even no logs about such setup

Reply:
should we then remove `in mercator mode` from the changelog then?

Reply:
@kiryldz well but if the api really does nothing when globe is turned on - why not mention it?

Reply:
@yunikkk I totally agree - that's why imo we should be consistent: either mention it here **and** in javadoc _or_ don't mention it anywhere.

Reply:
I see, than my suggestion is to mention everywhere =)

Reply:
lets remove `mercator mode`, its not mentioned in javadoc as well.

Reply:
just update changelog